runelite-client/src/main/java/net/runelite/client/plugins/itemstats/ItemStatChanges.java

```diff
@@ -110,12 +122,16 @@ private void init()
 		// Many of these retrieved from https://oldschool.runescape.wiki/w/Temporary_skill_drain
 		add(combo(food(11), dec(ATTACK, 2)), JUG_OF_WINE);
 		add(combo(food(14), dec(ATTACK, 3)), BOTTLE_OF_WINE);
+		add(combo(food(7), dec(ATTACK, 2)), HALF_FULL_WINE_JUG);
+		add(combo(dec(ATTACK, 3)), JUG_OF_BAD_WINE);
 		add(combo(3, food(5), new SimpleStatBoost(STRENGTH, true, perc(.05, 1)), new BoostedStatBoost(ATTACK, false, perc(.02, -3))), PREMADE_SGG, SHORT_GREEN_GUY, BRANDY, GIN, VODKA, WHISKY);
 		add(combo(3, food(7), new SimpleStatBoost(STRENGTH, true, perc(.05, 2)), new BoostedStatBoost(ATTACK, false, perc(.02, -3))), PREMADE_BLURB_SP, BLURBERRY_SPECIAL);
 		add(combo(3, food(5), new SimpleStatBoost(STRENGTH, true, perc(.05, 2)), new BoostedStatBoost(ATTACK, false, perc(.02, -3))), PREMADE_DR_DRAGON, DRUNK_DRAGON, PREMADE_CHOC_SDY, CHOC_SATURDAY);
 		add(combo(3, food(5), new SimpleStatBoost(STRENGTH, true, perc(.06, 1)), new BoostedStatBoost(ATTACK, false, perc(.02, -3))), PREMADE_WIZ_BLZD, WIZARD_BLIZZARD);
 		add(combo(3, food(3), new SimpleStatBoost(STRENGTH, true, perc(.04, 1)), new BoostedStatBoost(ATTACK, false, perc(.05, -3))), GROG);
 		add(combo(3, food(1), boost(STRENGTH, perc(.02, 1)), new BoostedStatBoost(ATTACK, false, perc(.06, -1))), BEER, BEER_7740);
+		add(combo(2, food(4), boost(STRENGTH, perc(.04, 2)), heal(ATTACK, perc(.1, -2))), BEER_TANKARD);
+		add(combo(2, food(15), boost(STRENGTH, perc(.1, 2)), heal(ATTACK, perc(.5, -4))), KEG_OF_BEER_3801 /* Non-quest version */);
```

Comment:
These impact 3 skills, not 2, so the `primaries` argument needs to be changed. (_hitpoints_, strength, and attack) Additionally, the skill drains are based on your boosted level, not your maximum level, so the attack drains should be `BoostedStatBoost`s instead of `heal`s. (see: https://oldschool.runescape.wiki/w/Temporary_skill_drain)

Reply:
I am a bit confused about what the `primaries` argument means. I don't quite understand the effect on the code that parameter has. If you look at the Jug of Wine, `primaries` is set to 1. Yet it changes both hitpoints and attack.

Reply:
It's used to check all the passed `SingleEffect`s to determine the positivity of the resulting `Combo`